dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -42,3 +42,7 @@ JAVAEESEC_WARNING_NO_REALM_NAME.useraction=If the realm name needs to be set, pl
 JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS=CWWKS1915E: The deployment for the {0} module in the {1} application failed because multiple HttpAuthenticationMechanism implementation are found. The list of classes is {2}. Most likely, this is an application packaging issue. Make sure that there is only one HttpAuthenticationMechanism implementation in each module.
 JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS.explanation=The application deployment failed because more than one HttpAuthenticationMechanism implementation are found. This condition is not allowed.
 JAVAEESEC_ERROR_MULTIPLE_HTTPAUTHMECHS.useraction=Make sure that there is only one HttpAuthenticationMechanism implementation for each module in the application.
+
+JAVAEESEC_WARNING_IDSTORE_CONFIG=CWWKS1916W: There was an error resolving the ''{0}'' configuration for the identity store. Ensure any EL expressions are resolvable. The value will be defaulted to ''{1}''.
+JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=The identity store configuration could not be resolved to a valid value.
```

Comment:
+JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=The identity store configuration cannot be resolved to a valid value.

Note: Can we tell the user how they can resolve the issue, such as adding a sentence that says how they can get valid values?

Reply:
Update:
Ensure that the EL expression and the result are valid and ensure any CDI beans that are used in expressions are resolvable.